travis-ci/travis-web
# Ep persist billing info

Persist billing information to prevent data loss on reload. This PR is related to #2160

I created a separate PR so this could be reviewed on its own, as the original PR was getting quite large.

Steps to test:

- Select a plan under subscription tab
- Enter Billing Info
- Attempt a refresh on second or third steps. 

Expected behaviour:
- Data should not be lost and step should be remembered.
